saleor/graphql/product/types/attributes.py

```diff
@@ -6,6 +6,7 @@
 from graphql_jwt.decorators import permission_required
 
 from ....product import models
+from ....product.utils.attributes import T_ASSIGNMENT_REL
```

Comment:
Maybe we could have own `typing` file for such structures? @maarcingebala  What do you think?

Reply:
Not sure if we have enough things to put into such file. I'm also not sure what is the best place to keep this one. But what I am sure, is that this name is a little bit scary as for our codebase ;) Maybe let's rename it to something like `AttributeAssignmentType`?